reV/supply_curve/supply_curve.py

```diff
@@ -259,6 +259,42 @@ def _parse_trans_table(trans_table):
 
         return trans_table
 
+    @staticmethod
+    def _map_trans_capacity(trans_sc_table):
+        """
+        Map SC gids to transmission features based on capacity. For any SC
+        gids with capacity > the maximum transmission feature capacity, map
+        SC gids to the feature with the largest capacity
+
+        Parameters
+        ----------
+        trans_sc_table : pandas.DataFrame
+            Table mapping supply curve points to transmission features.
```

Comment:
So this is after joining the SC agg table to the Trans "all cons" table right?

Reply:
yes, this is the fully agg_table + trans_features table merged on sc_point_gid